### ISSUE TITLE
Encountering install error on RHEL 8.3 +

### ISSUE THREAD

**Issue by justinkreft** [quipucords/quipucords#2086]
## Bug Description:
Running into a failed install error when installing QPC to RHEL 8.3 +. I have yet to successfully install QPC/Discovery  8.3 or 8.4. Other end users have confirmed that this is the error that is raised.

### Expected behavior:
Successful install

### Actual behavior:
~~~
FAILED - RETRYING: Ping discovery server to validate it is running and functional (23 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (22 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (21 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (20 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (19 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (18 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (17 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (16 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (15 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (14 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (13 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (12 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (11 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (10 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (9 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (8 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (7 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (6 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (5 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (4 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (3 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (2 retries left).
FAILED - RETRYING: Ping discovery server to validate it is running and functional (1 retries left).
fatal: [localhost]: FAILED! => {"attempts": 30, "changed": false, "content": "", "elapsed": 0, "msg": "Status code was -1 and not [200]: Request failed: <urlopen error [Errno 111] Connection refused>", "redirected": false, "status": -1, "url": "https://127.0.0.1:9443/api/v1/status"}

PLAY RECAP *********************************************************************
localhost                  : ok=42   changed=17   unreachable=0    failed=1    skipped=18   rescued=0    ignored=1 
~~~

### Steps to reproduce:
Install QPC/Discovery on a fresh 8.3 + RHEL OS

I can confirm that the GUI will not load at localhost:9443 but that the containers are up

~~~
CONTAINER ID  IMAGE                                                                    COMMAND               CREATED         STATUS             PORTS                  NAMES
ae6a1302d016  registry.redhat.io/rhel8/postgresql-96:latest                            run-postgresql        25 minutes ago  Up 25 minutes ago  0.0.0.0:9443->443/tcp  dsc-db
8ca3730cff4c  registry.redhat.io/discovery-tech-preview/discovery-server-rhel8:latest  /bin/bash /deploy...  26 minutes ago  Up 25 minutes ago  0.0.0.0:9443->443/tcp
~~~
___

## Acceptance Criteria:

Fix install issuse for RHEL 8.3 +

### Environment information:
RHEL 8.3 and 8.4

**Comment by bruno-fs** [comment]
I believe the changes merged on quipucords/qpc-tools#124 will solve this problem.

**Comment by justinkreft** [comment]
Closing Resolved with 1.0
